### DISCUSSION TITLE
splitting database .json file

### DISCUSSION THREAD

**Discussion by ejamshidiasl** [msiemens/tinydb#511] [Extensions]
hello cool programmers.
store each X rows of data (ex. 20000) to a file and then for search and update, go in files one by one until find and for insert, insert to last file.
and a function to check and rewrite to files that may have removed or updated data (this function must be manual. programmers can call it if they want).
in this way we can store non limited amount of data without performance issue

**Comment by msiemens** [comment]
Hey @ejamshidiasl, I think you can implement a custom storage that does this but in my view such a scheme would be beyond the scope of TinyDB 🙂
